ksqldb-engine/src/main/java/io/confluent/ksql/services/DefaultConnectClient.java

```diff
@@ -76,7 +78,7 @@
   private static final String STATUS = "/status";
   private static final String TOPICS = "/topics";
   private static final String VALIDATE_CONNECTOR = CONNECTOR_PLUGINS + "/%s/config/validate";
-  private static final int DEFAULT_TIMEOUT_MS = 5_000;
+  private static final int DEFAULT_TIMEOUT_MS = 30_000;
```

Comment:
I considered making this value configurable, if we think defaulting to 30 seconds in order to accommodate certain high latency environments is too extreme. OTOH, we already have so many server configs, maybe it doesn't make sense to add another. I could go either way on this one, might be leaning towards adding a config.

Reply:
Update: made the value configurable and set the default back to 5 ms. Users with in high latency environments can increase their timeouts via the new config.